create_workspace.md

```diff
@@ -0,0 +1,74 @@
+  - To set up the environment for STEMMUS-SCOPE, we only need to choose `SURF HPC Cloud` and `jupyter-workspace` environment with preferred quantity of cpu cores and memory (e.g. 2 cpu cores and 16 GB RAM).
+  - In step 5 "Options", remember to attach the storage as this is the only moment we can attach it to our workspace.
+- Log in to your workspace follwing this [documentation](https://servicedesk.surf.nl/wiki/display/WIKI/Log+in+to+your+workspace)).
+
+## Install Matlab Runtime
+To intall Matlab Runtime for all users, it is easy to use `ansible playbook`.
+An example `ansible-playbook` showing how to install matlab runtime is available [here](./install_matlab_runtime.yml). Please copy it in your jupyter-workspace and run the following command to download and install matlab runtime:
+
+```sh
+ansible-playbook install_matlab_runtime.yml
+```
+
+## Download the executable file of STEMMUS-SCOPE model
+The latest version of the executable file is available in the repo [`STEMMUS_SCOPE`](https://github.com/EcoExtreML/STEMMUS_SCOPE). We can clone this repo and copy the executable file to the shared storage:
+```py
+cd ~
+git clone https://github.com/EcoExtreML/STEMMUS_SCOPE.git
+cp STEMMUS_SCOPE/run_model_on_snellius/exe/STEMMUS_SCOPE ~/data/volume_2/
+```
+
+Note: currently `STEMMUS_SCOPE` is a private repo and therefore it is not allowed to clone it using `HTTPS`. We have to setup `SSH` connection for it.
+
+To generate a new ssh key, run the following command in your terminal:
+```sh
+ssh-keygen -t ed25519 -C "your_email@example.com"
+```
+Ensure the ssh-agent is running by checking:
+```sh
+eval "$(ssh-agent -s)"
+```
+If we can see `Agent pid xxxxx`, then you can add your SSH private key to the ssh-agent:
+```sh
+ssh-add ~/.ssh/id_ed25519
+```
+
+And finally you can add your public key to the github in `settings/SSH and GPG keys`.
+
+(For more details about generating ssh key and adding it to github, check this [documentation](https://docs.github.com/en/authentication/connecting-to-github-with-ssh/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent)).
+
+## Copy forcing data to the shared storage
+In order to execute the notebook and run the model, we need to copy all the forcing and boundary conditions data to the storage of SRC.
+
+If the data is stored on a server, we can simply connect the server via SSH in our jupyter-workplace and copy the data to the SRC storage. For instance, if we have our data stored on the project space of Snellius, we can open a terminal in the jupyter-workplace and type the following command to connect to the server via SSH:
+```sh
+sftp user@snellius.surf.nl
+```
+
+The server will ask for your password. After typing the correct password, you should be able to access the storage system on Snellius. Now you can navigate to the path of data and download the data to the storage of SRC using the following commands:
+
+```sh
+cd /path_to_data
```

Comment:
For future you it might be nice to be more explicit on the data location.

Reply:
@BSchilperoort is working on scripts to fill that directory at https://github.com/EcoExtreML/STEMMUS_SCOPE_Processing/blob/main/global_data/downloading_global_data.md